src/System.Xml.XmlSerializer/tests/Performance/XsPerformanceTest.cs

```diff
@@ -17,7 +17,12 @@ public class XsPerformanceTest
     {
         public static IEnumerable<object[]> SerializeMemberData()
```

Comment:
Yes if we can use the PerfTestConfig as parameter of MemberData then it would be nice but xunit performance doesn't work (https://github.com/Microsoft/xunit-performance/issues/124)